### ISSUE TITLE
consider how the balance of active/passive components over time, re: trainee fatigue

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/instructor-training#1402]
> By the 7th hour of the first day of training, I was no longer able to concentrate on the material or interact with others (as fatigue exacerbates autistic symptoms, and for me executive dysfunction in particular). This comment was mirrored in the Zoom chat by a few other participants.

> Lesson episodes which require 1:n interaction or verbal presentations at the end of an 8-hour day can also be exclusionary for autistic people. The final lesson episode of the first day involved presenting the first 90 seconds of a lesson to peers. Our instructor trainers (@jsteyn, @anenadic and Philipp Schäfer – thanks all!!) showed great flexiblilty in rearranging this part of the content to become the first exercise of the second day in response to feedback. 

_Originally posted by @sjmf in https://github.com/carpentries/instructor-training/issues/1395#issuecomment-1163278279_

**Comment by amanda-dartmouth** [comment]
I agree that Day 1 was very full and to end with the first lesson demonstration was difficult. I suggest moving it to Day 2. Take the worry and stress out of doing this on Day 1. Have all of the lesson demonstrations be on Day 2.
